### PR TITLE
FreeWheelSampleAppAutomationHook

### PR DESCRIPTION
**Pull request by bindumadhav**

### CODE REVIEW

File: FreewheelSampleApp/FreewheelSampleApp/AppDelegate.m [hunk 1]
```diff
@@ -12,9 +12,15 @@ @interface AppDelegate () <UINavigationControllerDelegate>
 @end
 
 @implementation AppDelegate
+AppDelegate *instance ;
```
Comment (lental) [inline FreewheelSampleApp/FreewheelSampleApp/AppDelegate.m]: Can you use a property here?

File: FreewheelSampleApp/FreewheelSampleApp/AppDelegate.h [hunk 2]
```diff
@@ -10,6 +10,8 @@
 @interface AppDelegate : UIResponder <UIApplicationDelegate>
 
 @property (strong, nonatomic) UIWindow *window;
++(AppDelegate*)getInstance;
+@property (nonatomic,assign) int count ;
```
Comment (lental) [inline FreewheelSampleApp/FreewheelSampleApp/AppDelegate.h]: space after count

Comment (bindumadhav) [inline FreewheelSampleApp/FreewheelSampleApp/AppDelegate.h]: sure, will remove the extra space after count

Comment (bindumadhav) [inline FreewheelSampleApp/FreewheelSampleApp/AppDelegate.h]: made the necessary changes to styling and space

**Comment by lental** [comment]
@Matthew-C says he provided a suggestion to use the SharedApplication.  going to hold this for after the release

**Comment by bindumadhav** [comment]
sounds good

**Pull request merged**